tcms/rpc/tests/test_testexecution.py

```diff
@@ -622,6 +624,31 @@ def test_update_with_no_perm(self):
                 self.execution_1.pk, {"stop_date": timezone.now()}
             )
 
+    def test_create_actual_duration(self):
+        test_execution = TestExecution()
+
+        test_execution.start_date = datetime.strptime("2020-05-05 09:00:00", "%Y-%m-%d %H:%M:%S")
+        test_execution.stop_date = datetime.strptime("2020-05-06 09:00:00", "%Y-%m-%d %H:%M:%S")
+
+        actual_duration_result = test_execution.stop_date - test_execution.start_date
+        self.assertEqual(test_execution.actual_duration,actual_duration_result)
```

Comment:
This test scenario is not related to the API methods thus needs to be moved into tcms/testcases/tests/test_models.py